test/components/live_file_input_test.exs

```diff
@@ -0,0 +1,102 @@
+defmodule Surface.Components.LiveFileInputTest do
+  use Surface.ConnCase, async: true
```

Comment:
I added a test to use the `:default_class` of the `LiveFileInput` component. That test requires that async must be set to `false`
```suggestion
  use Surface.ConnCase
```